### DISCUSSION TITLE
IBAX Geleceğin AVAX'ı mı?

### DISCUSSION THREAD

**Discussion by TheMunay** [IBAX-io/go-ibax#195] [Ideas]
IBAX şimdiden AVAX ile kıyaslanmaya başlıyor. Geleceğin projesi olarak öngörülüyor. Sizce bu duygusal düşünülerek abartılıyor mu yoksa  gerçeklik payı var mı :) Airdroptan alınacak sonuç da gösterecektir bence

**Comment by ghost** [comment]
var

**Reply by Denzmur** [reply]
Geleceğin avax si deniyor ama geleceği öngörmek zor .önceki deneyimlere göre olabilir ama henua birşey söylemek için erken ..su tokenler gelse az çok belli olacak

**Comment by Grekny** [comment]
gerçeklik payı şuan son derece az tamamen böyle olsun istedikleri için böyle söyleyip coini hypelıyorlar ama şuan için böyle belki ilerde gerçekten avaxla karşılaştırılabilir.

**Reply by sedat2855** [reply]
evet

**Comment by byblckd** [comment]
şuanlık hiçbir gerçekliği olmayan bir olay. Sadece insanları hype yönlendirme çabası başka birşey değil.Şuan değil sol avax gibi olması kendi ekosistemi olmayan herhangi bir listelenmiş tokena bile eş değer değil. Herşey zamanla yapacakları projelere bağlı olarak gerçekleşebilir

**Comment by Halo0642** [comment]
Bence çok sağlam bir proje,umarım herkes umduğunu kazanır

**Comment by con-quistador** [comment]
kıyaslamak hata olabilir proje tam da yeni trende hizmet edecek bir proje umarım daha iyi yerlere gider

**Comment by masumkurt** [comment]
Good

**Comment by HsMetto** [comment]
Bencede mantıklı bir proje metaversele birlikte kullanılacağını vaat ediyorlar eğer gerçekleştirebilirlerse iyi iş başarmış olacaklar

**Comment by Ahstnc** [comment]
Avax mı bilmem ama iyi yerlere gelir
